sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileWriteBuilder.scala

```diff
@@ -68,26 +61,8 @@ abstract class FileWriteBuilder(
     lazy val description =
       createWriteJobDescription(sparkSession, hadoopConf, job, paths.head, options.asScala.toMap)
 
-    val fs = path.getFileSystem(hadoopConf)
-    mode match {
-      case SaveMode.ErrorIfExists if fs.exists(path) =>
```

Comment:
Could you give me the pointer where handles this?

Reply:
e.g. https://github.com/apache/spark/blob/8b01473e8bffe349b1ed993b61420d7d68896cd8/sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala#L603